src/Symfony/Component/Config/Exception/FileLoaderLoadException.php

```diff
@@ -11,99 +11,7 @@
 
 namespace Symfony\Component\Config\Exception;
 
-/**
- * Exception class for when a resource cannot be loaded or imported.
- *
- * @author Ryan Weaver <ryan@thatsquality.com>
+/*
```

Comment:
this should start a real docblock: `/**`
and in order to help IDEs and composer detect the deprecation and the alias, it should be on a real class declaration:
```php
if (false) {
    /**
     * @deprecated since Symfony 4.2, use LoaderLoadException instead.
     */
    class FileLoaderLoadException extends LoaderLoadException
    {
    }
}

then a real deprecation should also be triggered (outside of the "if"):
`@trigger_error(..., E_USER_DEPRECATED);`